src/components/tables/PoolsTable/PoolsTable.vue

```diff
@@ -316,17 +321,13 @@ function iconAddresses(pool: Pool) {
       </template>
       <template #poolNameCell="pool">
         <div v-if="!isLoading" class="flex items-center py-4 px-6">
-          <div v-if="poolMetadata(pool.id)?.name" class="pr-2 text-left">
-            {{ poolMetadata(pool.id)?.name }}
-          </div>
-          <div v-else>
-            <TokenPills
-              :tokens="orderedPoolTokens(pool, pool.tokens)"
-              :isStablePool="isStableLike(pool.poolType)"
-              :selectedTokens="selectedTokens"
-              :pickedTokens="selectedTokens"
-            />
-          </div>
+          <PoolName
+            :poolIds="poolIds"
```

Comment:
you're passing this poolIds to use it as a cache key in the query so that we avoid making tons of queries when opening that list page, right?

Reply:
we are actually passing this poolIds to get the pools id of the pools that are being rendered on screen. 

```js
const { pools, isLoading, poolsIsFetchingNextPage, loadMorePools } = usePools(
  selectedTokens,
  poolsSortField
);
```

fetch 10 pools on the first render, then only when the user clicks on `load more` than 10 new pools fetched using `usePools`. What we need is to get the IDs from only the rendered pools  and search for their custom metadata